tools/solidityUpgrade/main.cpp

```diff
@@ -85,10 +85,12 @@ int main(int argc, char** argv)
 	catch (boost::exception const& _exception)
 	{
 		cerr << "Exception while processing input: " << boost::diagnostic_information(_exception) << endl;
+		return 2;
 	}
 	catch (...)
 	{
 		cerr << "Unknown exception while processing input: " << boost::current_exception_diagnostic_information() << endl;
+		return 2;
```

Comment:
Oh, so it was not returning an error status at all in case of an exception? Then I'd actually classify this bit as bugfix. We might even want to add another changelog entry just for that. The upgrade utility is not commonly used, but still, it's there and documented.